services:
  syncthing:
    container_name: syncthing-sean
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.syncthing-sean.rule=Host(`sean.syncthing.fzymgc.house`)"
      - "traefik.http.routers.syncthing-sean.entrypoints=websecure"
      - "traefik.http.routers.syncthing-sean.tls=true"
      - "traefik.http.routers.syncthing-sean.tls.certresolver=letsencrypt"
      - "traefik.http.services.syncthing-sean.loadbalancer.server.port=20910"
    hostname: syncthing-sean
    image: docker.io/syncthing/syncthing:latest@sha256:2a1eaba4d4dd5f63994039c8506c19518d87fb964953c60c310588e52034d220
    environment:
      UID: 568
      GID: 568
      GROUP_ID: 568
      HOME: /var/syncthing
      PGID: 568
      PUID: 568
      STGUIADDRESS: 0.0.0.0:20910
      STNOUPGRADE: true
      UMASK: 002
      UMASK_SET: 002
      USER_ID: 568
      TZ: US/Eastern
    volumes:
      - /mnt/main/homes/sean:/mnt/homes/sean
      - /mnt/apps/syncthing-sean/config:/var/syncthing
    restart: unless-stopped
    network_mode: host
    healthcheck:
      test: curl -fkLsS -m 2 127.0.0.1:20910/rest/noauth/health | grep -o --color=never OK || exit 1
      interval: 1m
      timeout: 10s
      retries: 3
